micro-rdk/src/common/conn/viam.rs

```diff
@@ -523,6 +523,22 @@ where
             {}
         }
 
+        let _sntp: Box<_>;
```

Comment:
I'm a bit reluctant to add more to the ViamServer struct, though it might be cleaner in the end. That or have it as a conditional global.  How it is currently would need to be `Box<dyn Any>` to pass native CI though.